scripts/start/submit.R

```diff
@@ -19,7 +19,7 @@
 
 ############## Define function: runsubmit #########################
 
-submit <- function(cfg, restart = FALSE) {
+submit <- function(cfg, restart = FALSE, stopOnDeleteError = TRUE) {
```

Comment:
the submit function must tolerate that the output subfolder already exists if the slurm job should continue writing the log there.